rust/ql/lib/codeql/rust/controlflow/internal/Completion.qll

```diff
@@ -94,6 +94,8 @@ class BooleanCompletion extends ConditionalCompletion, TBooleanCompletion {
     )
     or
     exists(Expr parent | this.isValidForSpecific0(parent) |
+      e = parent.(ParenExpr).getExpr()
```

Comment:
Is the following on line 74 still needed `isBooleanConstant(n.(ParenExpr).getExpr(), value)` ?

Reply:
Yes, it's still needed.